### PR TITLE
Track which users initiate which tasks

### LINKED ISSUE #43: Get all tasks for a given user
The `task` endpoint is currently able to return all tasks that a given organization initiated. The feature request here is to extend this to getting the tasks for a given user.

This feature requires a new field to be added in the database.

- [x] Add column to database and fill it when task is created
- [x] Add filter option to the `task` endpoint
- [x] Implement in client
- [x] Implement in UI

### PR DESCRIPTION
**Pull request by bartvanb**
Fix #43 (except for the UI part which is a separate PR in the UI repo)

### CODE REVIEW

File: vantage6-client/vantage6/client/__init__.py [hunk 1]
```diff
@@ -1509,8 +1509,9 @@ def get(self, id_: int, include_results: bool = False) -> dict:
             return self.parent.request(f'task/{id_}', params=params)
 
         @post_filtering()
-        def list(self, initiator: int = None, collaboration: int = None,
-                 image: str = None, parent: int = None, run: int = None,
+        def list(self, init_org_id: int = None, init_user_id: int = None,
```
Comment (frankcorneliusmartin) [inline vantage6-client/vantage6/client/__init__.py:1512]: I do find this a bit incosistent. We never specified that it needs to be an `_id`. So either we need to change that all enpoints or I would not start using it here.

Comment (frankcorneliusmartin) [reply vantage6-client/vantage6/client/__init__.py:1512]: Same goes for the user

File: vantage6-node/vantage6/node/server_io.py [hunk 2]
```diff
@@ -134,14 +134,14 @@ def set_task_start_time(self, id: int):
             "started_at": datetime.datetime.now().isoformat()
         })
 
-    def patch_results(self, id: int, initiator_id: int, result: dict):
+    def patch_results(self, id: int, init_org_id: int, result: dict):
```
Comment (frankcorneliusmartin) [inline vantage6-node/vantage6/node/server_io.py:137]: Internally this is not an issue

**Review by frankcorneliusmartin — COMMENTED** [review]
Could you also add the init user and org in the __repr__ function of the `model/task`?
Other than that it looks good to me!

**Review by frankcorneliusmartin — CHANGES_REQUESTED** [review]
Remove logging from the `only_for` decorator, as mentioned

File: vantage6-client/vantage6/client/__init__.py (hunk 1 above)
Comment (bartvanb) [reply vantage6-client/vantage6/client/__init__.py:1567]: Made this consistent

File: vantage6-node/vantage6/node/server_io.py (hunk 2 above)
Comment (bartvanb) [reply vantage6-node/vantage6/node/server_io.py:137]: Left this the same - as it is not an issue

**Review by frankcorneliusmartin — APPROVED** [review]
lgtm

**Pull request merged**